src/modules/dashboard/components/Dialogs/CancelStreamingDialog/CancelStreamingForm.tsx

```diff
@@ -0,0 +1,287 @@
+          <FormattedMessage {...MSG.description} />
+        </p>
+      </DialogSection>
+      <DialogSection appearance={{ theme: 'sidePadding' }}>
+        <h4 className={styles.dialogSectionTitle}>
+          <FormattedMessage {...MSG.detailsTitle} />
+        </h4>
+        <div
+          className={classNames(
+            styles.row,
+            styles.rowAlt,
+            styles.userContainer,
+          )}
+        >
+          <InputLabel
+            label={MSG.to}
+            appearance={{
+              direction: 'horizontal',
+            }}
+          />
+          <div className={styles.userAvatarContainer}>
+            <UserAvatar address={walletAddress} size="xs" notSet={false} />
+            <div className={styles.userName}>
+              <UserMention username={username || ''} />
+            </div>
+          </div>
+        </div>
+        <div
+          className={classNames(
+            styles.row,
+            styles.rowAlt,
+            styles.userContainer,
+          )}
+        >
+          <InputLabel
+            label={MSG.starts}
+            appearance={{
+              direction: 'horizontal',
+            }}
+          />
+          <span className={styles.value}>
+            <FormattedMessage {...MSG.startsValue} />
+          </span>
+        </div>
+        <div
+          className={classNames(
+            styles.row,
+            styles.rowAlt,
+            styles.userContainer,
+          )}
+        >
+          <InputLabel
+            label={MSG.ends}
+            appearance={{
+              direction: 'horizontal',
+            }}
+          />
+          <span className={styles.value}>
+            <FormattedMessage {...MSG.endsValue} />
+          </span>
+        </div>
+      </DialogSection>
+      {fundingSourcesMock && (
+        <ul>
+          {fundingSourcesMock.map(({ filteredDomainId, ...props }, index) => (
+            <li key={props.id}>
+              <DialogSection appearance={{ theme: 'sidePadding' }}>
+                <FundingSourceItem
+                  {...props}
+                  {...{ colony, index }}
+                  filteredDomainId={filteredDomainId?.toString()}
+                  isMultiple={fundingSourcesMock.length > 1}
+                />
+              </DialogSection>
+            </li>
+          ))}
+        </ul>
+      )}
+      <DialogSection appearance={{ theme: 'sidePadding' }}>
+        <div className={styles.annotations}>
+          <Annotations
+            label={MSG.textareaLabel}
+            name="annotation"
+            maxLength={90}
+          />
+        </div>
+      </DialogSection>
+      <DialogSection appearance={{ align: 'right', theme: 'footer' }}>
+        <Button
+          appearance={{
+            theme: 'primary',
+            size: 'large',
+          }}
+          style={{
+            width: styles.buttonWidth,
+          }}
+          autoFocus
+          text={{ id: 'button.submit' }}
```

Comment:
I also noticed the button text is different from the one in the designs, it should be "Create Motion":
<img width="330" alt="Screen Shot 2022-10-04 at 11 45 54 PM" src="https://user-images.githubusercontent.com/71563622/193936808-b048fb75-2a93-4269-a6dd-4bc14a630652.png">

And when the streaming action is toggled it should be:
<img width="232" alt="Screen Shot 2022-10-04 at 11 46 43 PM" src="https://user-images.githubusercontent.com/71563622/193937034-74830ddb-f4de-4542-93a2-2cef9aa722c0.png">

Reply:
i have changed the button text, but i can not find in the design the second option when it comes to the submit buttons, here is the issue #3890 and on the design there is a separate modal with similar (back and cancel actions - https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5779%3A332781), can you show the place where you have found those back and force buttons design?

Reply:
@arrenv, curious if we need to add the Force Change text on toggle to this modal? Since I am assuming this works as any other action in the DApp?

Reply:
We don't need to add the "Force Change" here. I don't mind it from a clarity perspective, and is something we should consider in future. However, we should be consistent with the rest of the DApp for now and just have the greenish button with "Continue". @Julianahlk @joannasia

Reply:
> DApp for now and just have the greenish button with "Continue". @Julianahlk @joanna

Ok, so just to be super clear, for the cancel modal we won't be using the "Create Motion" text like it it shown in the [designs](figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5779%3A332781) and instead it should say "Continue"?

Reply:
@arrenv FYI it has cancel label atm

Reply:
@arrenv, I was seeing this: 
<img width="585" alt="Screen Shot 2022-10-06 at 7 28 26 PM" src="https://user-images.githubusercontent.com/71563622/194379603-2373339f-da9f-4240-8438-9fffc5d7d730.png">
Which is why I wanted to double check.

Reply:
It should have "Continue" for Force, and "Create Motion" for creating a Motion. As it is elsewhere.